### PR TITLE
tools: add cqlsh shortcut

### PR DESCRIPTION
**Pull request by avikivity**
Add bin/cqlsh as a shortcut to tools/cqlsh/bin/cqlsh, intended for developers.

### CODE REVIEW

File: bin/cqlsh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Copyright (C) 2023-present ScyllaDB
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+here=$(dirname "$0")
+exec "$here/../tools/cqlsh/bin/cqlsh" "$@"
```
Comment (tchaikov) [inline bin/cqlsh:7]: how about having a tiny script in `/scripts`, so we can just source it for setting up the alias / functions or `$PATH` ? developers can even source this script in their `$HOME/.bash_profile`.

Comment (avikivity) [reply bin/cqlsh:7]: I don't mind it. Personally I prefer stateless environments (that don't rely on me to source some script), but it's okay to have one for people who want it.

Comment (denesb) [reply bin/cqlsh:7]: I also prefer stateless environments. I have over 10 ScyllaDB trees checked out on my work machine. Which one do I source the env script from? :)

Comment (tchaikov) [reply bin/cqlsh:7]: So do I. Please refer to virtualenv.

**Comment by avikivity** [comment]
@scylladb/scylla-maint ping. I don't think @tchaikov's desire for a shell alias conflicts with this patch.

**Review by nyh — APPROVED** [review]
Looks good. By the way when the script is so trivial, it could have been replaced by just a symbolic link (I think git is capable of saving those, isn't it?)
If you do have a script, I would have added a comment saying what is the appropriate process to build this tools/cqlsh/bin/cqlsh - it doesn't come out of the repository. Even better than a comment, you could "test -x tools/cqlsh/bin/cqlsh" and if missing, print a message on how to build it.
But of course, such improvements can come later.
P.S. we could have the same shortcut for dbuild, so I can type bin/dbuild instead of tools/toolchain/dbuild. And also bin/seastar-addr2line, instead of seastar/scripts/seastar-addr2line and specifying the "-e" manually.

**Pull request closed without merging**